addons/event/static/src/views/event_slot_calendar/event_slot_calendar_model.js

```diff
@@ -8,13 +9,25 @@ export class EventSlotCalendarModel extends CalendarModel {
      * Save slot date and hours from selected datetimes
      */
     buildRawRecord(partialRecord, options = {}) {
-        const rawRecord = super.buildRawRecord(partialRecord, options)
+        const rawRecord = super.buildRawRecord(partialRecord, options);
         rawRecord["date"] = serializeDate(partialRecord.start);
         rawRecord["start_hour"] = partialRecord.start.hour + partialRecord.start.minute / 60;
-        rawRecord["end_hour"] = partialRecord.end.hour + partialRecord.end.minute / 60;
+        if (partialRecord.end) {
+            rawRecord["end_hour"] = partialRecord.end.hour + partialRecord.end.minute / 60;
+        }
         return rawRecord;
     }
 
+    /**
+     * @override
+     * Save selected date in context defaults for quick create form.
+     */
+    makeContextDefaults(rawRecord) {
+        const context = super.makeContextDefaults(rawRecord);
+        context["default_date"] = rawRecord["date"];
+        return context;
+    }
+
```

Comment:
Again, not sure what this does. Do we need the date in the quick create form? What exactly would be the quick create form in this case? (sorry, all those views make it rather confusing ^^')